styleguide/components/31-molecules/teaser/teaser--teaser-wide.twig

```diff
@@ -0,0 +1,66 @@
+<li class="teaser {{ teaser_type }}">
```

Comment:
... @meirege, let's continue the `teaser` prefix of the template variables here: 

It's just a consistency remark. We don't prefix the variables in any other component, so why would we do it here? Just legacy that should be refactored and since we're refactoring the whole teaser-component, we should do it right away, no? It's basically a search+replace in the codebase :shrug:

Reply:
Done.